2024-07-04 15:41:59,515 [INFO] root tankworker.py:153	Log file created
2024-07-04 15:41:59,515 [INFO] root tankworker.py:157	Logging handler <FileHandler /var/loadtest/tank_errors.log (WARNING)> added
2024-07-04 15:41:59,515 [INFO] root tankworker.py:157	Logging handler <StreamHandler <stdout> (INFO)> added
2024-07-04 15:41:59,515 [INFO] root tankworker.py:157	Logging handler <StreamHandler <stderr> (NOTSET)> added
2024-07-04 15:41:59,526 [INFO] root tankworker.py:69	Created a folder for the test. /var/loadtest/logs/2024-07-04_15-41-59.454208
2024-07-04 15:41:59,528 [INFO] yandextank.core.tankcore tankcore.py:254	Configuring plugins...
2024-07-04 15:41:59,529 [INFO] yandextank.core.tankcore tankcore.py:205	Loading plugins...
2024-07-04 15:41:59,529 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: autostop
2024-07-04 15:41:59,529 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.Autostop
2024-07-04 15:41:59,530 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: console
2024-07-04 15:41:59,530 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.Console
2024-07-04 15:41:59,530 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: json_report
2024-07-04 15:41:59,530 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.JsonReport
2024-07-04 15:41:59,533 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: phantom
2024-07-04 15:41:59,533 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.Phantom
2024-07-04 15:41:59,534 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: rcheck
2024-07-04 15:41:59,534 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.ResourceCheck
2024-07-04 15:41:59,534 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: shellexec
2024-07-04 15:41:59,534 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.ShellExec
2024-07-04 15:41:59,535 [INFO] yandextank.core.tankcore tankcore.py:215	Trying to import module: rcassert
2024-07-04 15:41:59,535 [INFO] yandextank.core.tankcore tankcore.py:216	Path: yandextank.plugins.RCAssert
2024-07-04 15:42:00,591 [WARNING] yandextank.common.util util.py:569	Using phantom.port option is deprecated. Use phantom.address=[address]:port instead
2024-07-04 15:42:00,592 [INFO] yandextank.common.util util.py:578	Testing connection to resolved address 192.168.15.36 and port 443
2024-07-04 15:42:00,595 [INFO] yandextank.plugins.Phantom.utils utils.py:277	Resolved partner.qatl.ru into 192.168.15.36:443
2024-07-04 15:42:00,595 [INFO] yandextank.stepper.main main.py:197	Configuring StepperWrapper...
2024-07-04 15:42:00,597 [INFO] yandextank.stepper.main main.py:336	Making stpd-file: /var/loadtest/ammo.stpd
2024-07-04 15:42:00,599 [ERROR] yandextank.stepper.config config.py:107	active sources: {'uris': ['/api/content/swagger/v1/swagger.json'], 'ammo_file': 'ammo.txt'}
2024-07-04 15:42:00,601 [ERROR] yandextank.common.util util.py:815	Exception occurred:
<class 'yandextank.stepper.module_exceptions.StepperConfigurationError'>: Exactly one of sources ['uris', 'ammo_file'] should be specified, but found ['uris', 'ammo_file'] in config.
  File "/usr/local/lib/python3.8/dist-packages/yandextank/core/tankworker.py", line 70, in run
    self.core.plugins_configure()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/core/tankcore.py", line 262, in plugins_configure
    plugin.configure()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/plugins/Phantom/plugin.py", line 67, in configure
    self.phantom_config = self.phantom.config_file

  File "/usr/local/lib/python3.8/dist-packages/yandextank/plugins/Phantom/utils.py", line 106, in config_file
    self._config_file = self.compose_config()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/plugins/Phantom/utils.py", line 114, in compose_config
    streams_config += stream.compose_config()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/plugins/Phantom/utils.py", line 288, in compose_config
    self.stepper_wrapper.prepare_stepper()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/main.py", line 264, in prepare_stepper
    self.__make_stpd_file()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/main.py", line 337, in __make_stpd_file
    stepper = Stepper(

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/main.py", line 59, in __init__
    self.af = AmmoFactory(ComponentFactory(**kwargs))

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/main.py", line 32, in __init__
    self.ammo_generator = factory.get_ammo_generator()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/config.py", line 125, in get_ammo_generator
    ammo_gen = self._get_ammo_generator()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/config.py", line 113, in _get_ammo_generator
    self._check_exactly_one_source_specified()

  File "/usr/local/lib/python3.8/dist-packages/yandextank/stepper/config.py", line 108, in _check_exactly_one_source_specified
    raise StepperConfigurationError(

2024-07-04 15:42:00,602 [INFO] yandextank.common.util util.py:816	Trying to clean up
2024-07-04 15:42:00,602 [INFO] yandextank.core.tankcore tankcore.py:630	Releasing lock: /var/lock/lunapark_t73d4e0g.lock
2024-07-04 15:42:00,613 [INFO] yandextank.core.tankcore tankcore.py:405	Artifacts dir: /var/loadtest/logs/2024-07-04_15-41-59.454208
2024-07-04 15:42:00,614 [INFO] yandextank.core.tankcore tankcore.py:461	File not found to collect: validation_error.yaml
2024-07-04 15:42:00,615 [INFO] yandextank.core.tankcore tankcore.py:466	File already exists: /var/loadtest/logs/2024-07-04_15-41-59.454208/answ_826gir8m.log
2024-07-04 15:42:00,617 [INFO] yandextank.core.tankcore tankcore.py:466	File already exists: /var/loadtest/logs/2024-07-04_15-41-59.454208/phout_5_3icyxf.log
2024-07-04 15:42:00,621 [INFO] yandextank.core.tankcore tankcore.py:466	File already exists: /var/loadtest/logs/2024-07-04_15-41-59.454208/phantom_stat_vtvg3bzx.log
2024-07-04 15:42:00,623 [INFO] yandextank.core.tankcore tankcore.py:466	File already exists: /var/loadtest/logs/2024-07-04_15-41-59.454208/phantom_u73f62ys.log
